app/controllers/api/v2/base_controller.rb

```diff
@@ -13,7 +13,7 @@ class BaseController < Api::BaseController
 
       def_param_group :pagination do
         param :page, :number, :desc => N_("Page number, starting at 1")
-        param :per_page, :number, :desc => N_("Number of results per page to return")
+        param :per_page, :number, :desc => N_("Number of results per page to return, 'all' to return all results")
```

Comment:
If it's still a `:number` "all" won't pass validation?

Reply:
This one: https://github.com/Apipie/apipie-rails/blob/master/lib/apipie/validator.rb#L435

Reply:
It should fail, but it's not failing for me :thinking:
We can probably just do regexp `/\A([1-9]\d*|all)\Z$/` validation though?

Reply:
it certainly will fail in apypie: https://github.com/Apipie/apypie/blob/master/apypie/action.py#L93